lib/housing_listing_scraper/padmapper_scraper.rb

```diff
@@ -22,13 +22,17 @@ def get_listing_URLs_from_search_html(search_html)
     end
 
     def parse_listing_html(listing_html)
-        
+
         doc = Nokogiri::HTML(listing_html)
         description = doc.at_css("pre").text
         address = doc.at("meta[name='place:street_address']")['content']
         zip_code = doc.at("meta[name='place:postal_code']")['content']
         latitude = doc.at("meta[name='place:location:latitude']")['content']
         longitude = doc.at("meta[name='place:location:longitude']")['content']
         return ListingInfo.new(description, address, zip_code, latitude, longitude)
-    end 
-end
\ No newline at end of file
+    end
+
+    def get_site_name()
+        return "Paddmaper"

```

Comment:
Would this (along with the search url) make more sense as attributes stored in the database as part of the scrapers table?